app/routes/client.py

```diff
@@ -183,24 +192,28 @@ def event_archive(index):
 
     :param int index: The page to fetch
     """
-    index = int(index)
     if index <= 0:
         return redirect(url_for('.events'))
 
+    # Get all events that occur on this page or on subsequent pages
     today = date.today()
-    last_sunday = datetime.combine(today - timedelta(days=today.weekday()+7),
-                                   datetime.min.time())
+    events = (Event.objects(published=True, end_date__lt=today)
+                   .order_by('-start_date')
```

Comment:
Yeah I didn't know of the '-' beforehand. Anyway I think in general more documentation is better. But since this is kind of trivial, I think an inline explanation fine.

Maybe to avoid a documentation spiral we can always have someone looking through our docs and making sure things aren't being repeated/are useful? Also, outside of the readme is there more documentation anywhere?